### ISSUE TITLE
Builder for build_runner isn't implemented

### ISSUE THREAD

**Issue by josh-tangelent** [DavBfr/pubspec_extract#2]
When using build_runner,  the builder cannot be found as it appears to be unimplemented as of version 2.0.3 of pubspec_extract.

When build_runner is running I get the following error:

```
[INFO] Generating build script...
[INFO] Generating build script completed, took 561ms

[INFO] Precompiling build script......
[WARNING] .dart_tool/build/entrypoint/build.dart:16:48: Error: Undefined name 'pubspecBuilder'.
      r'pubspec_extract:pubspec_extract', [_i3.pubspecBuilder], _i1.toRoot(),
                                               ^^^^^^^^^^^^^^
[INFO] Precompiling build script... completed, took 10.1s

[SEVERE]

```

As temporary solution I commented out the builders section in the build.yaml file so that build_runner ignores searching for the builder.

**Comment by DavBfr** [comment]
Published as version 2.0.4

**Comment by josh-tangelent** [comment]
Awesome, thanks David!  We use this package in one of our projects and find it very useful.
